modules/n1ql/pages/n1ql-language-reference/typefun.adoc

```diff
@@ -199,6 +227,14 @@ Returns True if expression is a string, otherwise returns MISSING, NULL or false
 
 === Description
 
+Type functions perform operations that check or convert expressions.
+
+=== Arguments
+
+expression:: [Required] The expression to check.
+
+=== Return Value
+
 Returns True if expression is a string, otherwise returns MISSING, NULL or false.
 
 === Examples
```

Comment:
The result of the example is still wrong. Look at the description of this function: we would expect that it will return `true` for `string`; `null` for `null`; and `false` for everything else.